client/payment-request/style.scss

```diff
@@ -46,13 +60,13 @@
 			background-color: #3c4043;
 		}
 		&:focus {
-			box-shadow: #5f6368 0 1px 1px 0, #5f6368 0 1px 3px;
 			outline: 0;
 		}
```

Comment:
Same as above ^, we need to include the `box-shadow` at the very least, and maybe allow the outline too.

Reply:
Stripe also removes the outline property. I guess the best here is to be compliant with Stripe's standards. Using the same `background-color` as in `:hover` fixes this.

![image](https://user-images.githubusercontent.com/5509901/116767290-a386a200-aa05-11eb-9ea6-7fb5abcc9de8.png)